Sources/PathBuilder/LibraryContent.swift

```diff
@@ -0,0 +1,36 @@
+import SwiftUI
+import DeveloperToolsSupport
+
+@available(macOS 11.0, *)
+struct LibraryContent: LibraryContentProvider {
+    var views: [LibraryItem] {
+        LibraryItem(
+            Path {
+                Move(to: CGPoint(x: 0, y: 0))
+                // ...
+                Close()
+            }
```

Comment:
Compilation is failing in current production version of Xcode (14.3.1). In Xcode 15 beta 4 this compiles.

Reply:
<img width="1512" alt="Screenshot 2023-07-21 at 17 18 02" src="https://github.com/mkj-is/PathBuilder/assets/708312/e512b27c-bf1d-4e34-8bf3-d2b7f48fb79e">

Reply:
This weird. I compiled this with no issue on Xcode 14.2. My mac is too old too upgrade to the latest version.

Reply:
No sure if removing the embedded code in the trailing closure will work.

Reply:
This fixes the warning: https://github.com/mkj-is/PathBuilder/pull/8/files#r1270831463